app/serializers/api/shop_relatives_serializer.rb

```diff
@@ -0,0 +1,17 @@
+class Api::ShopRelativesSerializer < ActiveModel::Serializer
```

Comment:
relatives are producers right?
you are creating a new concept "relatives", is it really necessary?

maybe it does not need to be specific to the relatives, maybe this can be simply called EnterpriseThinSerializer, it's just a thin representation of an enterprise, it's not related to relationships between enterprises as the relatives implies.
I'd call this EnterpriseThinSerializer

Reply:
regarding the names of the serializers:

option 1 "named serializers" would be to:
- rename Api::ShopSerializer to Api::EnterpriseShopfrontSerializer
- rename Api::ShopRelativesSerializer to Api::EnterpriseThinSerializer
- rename Api::ShopsSerializer to Api::EnterpriseWithIconsSerializer

option 2 "for serializers" would be to:
- rename Api::ShopSerializer to Api::EnterpriseSerializerForShopfront
- rename Api::ShopRelativesSerializer to Api::EnterpriseSerializerForRelatives
- rename Api::ShopsSerializer to Api::EnterpriseSerializerForMaps

option 3 "mix" would be to:
- rename Api::ShopSerializer to Api::EnterpriseSerializerForShopfront
- rename Api::ShopRelativesSerializer  to Api::EnterpriseThinSerializer
- rename Api::ShopsSerializer to Api::EnterpriseWithIconsSerializer

I vote to keep everything in the Enterprise namespace and not create yet another domain concept called Shop, including the api controller.

I think this is an important API design decision because we should then use the same approach everywhere.
ping @sauloperez  @mkllnk @kristinalim 
What do you guys think?